docs/src/configuration/artifacts.md

```diff
@@ -0,0 +1,37 @@
+# Artifacts & `cargo-dist`
+
+oranda has first-class support for linking to generated artifacts and installers, using [`cargo-dist`].
```

Comment:
```suggestion
oranda has first-class support for handling releases generated by [`cargo-dist`]. It can even detect the user's platform and recommend the best installer/archive to use!
```